src/saveReportAsJson.js

```diff
@@ -0,0 +1,9 @@
+import fs from 'fs';
+import { shell } from 'electron';
+
+export default async function makeJSON(data, savePath) {
+  fs.writeFile(savePath, data, (error) => {
+    if (error) throw error;
```

Comment:
Should we show a modal to user in case the json fails?

Reply:
@ankitsinghaniyaz I assume it means writeFile fails (and an error occurs)?

Reply:
This is not a priority but could be a good thing. For example if the download file fails if the user is trying to save where he doesn't have write permission etc. An error could help the user understand why the download didn't happened and make it easy, but can be skipped for now.

Reply:
Yes that would be a good thing, we will do this.